kpc/models.py

```diff
@@ -9,7 +9,6 @@ class Licensee(models.Model):
     within the United States
     """
     name = models.CharField(max_length=256)
-    primary_contact = models.CharField(max_length=32)
```

Comment:
Just to confirm, no need to try to migrate any of this data, right?

Reply:
Yep, we're still in green pastures, thanks for double checking!